component/src/main/java/me/androidbox/component/general/TaskButton.kt

```diff
@@ -8,6 +8,7 @@ import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
```

Comment:
Are you using this import?

Reply:
@MKiperszmid Removed the unused import. Normally, I have a spotless check to would remove all unused imports but not had time to set it up in this project.